tests/integration/conftest.py

```diff
@@ -407,26 +430,31 @@ def missing_os_release_package_workaround(shell):
     yield
 
     os_to_pkg_mapping = {
-        "oracle-7": "oracle*-release-el7",
-        "oracle-8": "oraclelinux-release-el8",
-        "centos-7": "centos-release",
-        "centos-8": "centos-linux-release",
-        "alma-8": "almalinux-release",
-        "rocky-8": "rocky-release",
+        "oracle-7": ("oraclelinux-release-el7", "oraclelinux-release"),
+        "oracle-8": ("oraclelinux-release-el8", "oraclelinux-release"),
+        "centos-7": ("centos-release",),
+        "centos-8": ("centos-linux-release",),
+        "alma-8": ("almalinux-release",),
+        "rocky-8": ("rocky-release",),
     }
 
     # Run only for non-destructive tests.
     # The envar is added by tmt and is defined in main.fmf for non-destructive tests.
-    if "INT_TESTS_NONDESTRUCTIVE" in os.environ:
+    if "C2R_TESTS_NONDESTRUCTIVE" in os.environ:
         os_name = SYSTEM_RELEASE_ENV.split("-")[0]
         os_ver = SYSTEM_RELEASE_ENV.split("-")[1]
         os_key = f"{os_name}-{os_ver[0]}"
 
-        system_release_pkg = os_to_pkg_mapping.get(os_key)
+        system_release_pkgs = os_to_pkg_mapping.get(os_key)
 
-        rpm_output = shell(f"rpm -qa {system_release_pkg}").output
-        if "not installed" in rpm_output:
-            shell(f"yum install -y --releasever={os_ver} {system_release_pkg}")
+        for pkg in system_release_pkgs:
+            rpm_output = shell(f"rpm -q {pkg}").output
+            if "not installed" in rpm_output:
+                shell(f"yum install -y --releasever={os_ver} {pkg}")
```

Comment:
If you don't want to rely on string checking, you could replace it by: 

```suggestion
            if shell(f"rpm -q {pkg}").returncode:
                shell(f"yum install -y --releasever={os_ver} {pkg}")
```

Or, to prevent undesired outcomes (if that ever happen): 

```suggestion
            rpm_output = shell(f"rpm -q {pkg}").output
            if "not installed" in rpm_output.lower():
                shell(f"yum install -y --releasever={os_ver} {pkg}")
```

Reply:
Checking for return code makes much more sense